packages/react-search-ui/src/containers/Facet.js

```diff
@@ -12,6 +12,8 @@ function findFacetValueInFilters(name, filters, filterType) {
   return filter.values;
 }
 
+const accentFold = (str = '') => str.normalize('NFD').replace(/[\u0300-\u036f]/g, "");
```

Comment:
Please use double quotes.

There should be a pre-commit hook that runs prettier and catches this, not sure what happened.

Reply:
The current eslint configuration not catching "quotes" issues. The following rule is required to properly catch "quotes" issues.

`"quotes": [2, "double", "avoid-escape"]`

![image](https://user-images.githubusercontent.com/12813750/58652077-62c11280-8330-11e9-932b-d9346e584da4.png)

Reply:
@m-sureshraj Ah ok, that's supposed to be caught by the prettier plugin for eslint. Not sure why it isn't working.